courses/free-courses-ta.md

```diff
@@ -86,10 +88,17 @@ CSE Tamila by Eezytutorials
 * [Introduction to Machine Learning(Tamil)](https://www.youtube.com/playlist?list=PLyqSpQzTE6M-9thAeyB2mRFYvvW8AWxXX) - IIT Madras NPTEL
 * [Machine Learning in Tamil](https://www.youtube.com/playlist?list=PLJtSFa-YIedYu2QfQaHJJBLT096RxtMHD) - Majaa Matrix
 * [Machine Learning in Tamil](https://youtube.com/playlist?list=PL5itdT07Pm8wxRaPWljPntnBmnOs4ExDM) - Nithya Duraisamy
+* [Machine Learning with Python and R](https://www.youtube.com/watch?v=ucvXw1XUAy0&list=PL-1QQC56x1gEgj8C4L2hw5orryqgdnuoP) - Data Science Alive
```

Comment:
for example use https://www.youtube.com/playlist?list=PL-1QQC56x1gEgj8C4L2hw5orryqgdnuoP
do the same with the others

Reply:
Okay 👍🏽